jboss_wildfly/tox.ini

```diff
@@ -6,7 +6,10 @@ envlist =
     py37
 
 [testenv]
+description =
+    py37: e2e ready
```

Comment:
Doesn't it support py2?

Reply:
Well it doesn't use python, so we don't really need to run it for multiple versions.